data/actionsmap/yunohost.yml

```diff
@@ -1384,3 +1384,28 @@ hook:
                 -d:
                     full: --chdir
                     help: The directory from where the script will be executed
+
+#############################
+#          Journals         #
+#############################
+journals:
+    category_help: Manage debug journals
+    actions:
+
+        ### journals_list()
+        list:
+            action_help: List journals
+            api: GET /journals
+            arguments:
+               -l:
+                    full: --limit
+                    help: Maximum number of journals per categories
+                    type: int
+
+        ### journals_display()
+        display:
+            action_help: List journals
+            api: GET /journals/<file_name>
```

Comment:
How to display the journal of an app from the APPID ? 
It could be useful to let the user access it from the webadmin app page.